### PR TITLE
docs: improve readme

### PR DESCRIPTION
**Pull request by crondinini**
### Overview

Addresses items raised in #181 and also adds a bit of instruction for running it locally. 

It also attempts to fix the [issue labeler error](https://github.com/celo-org/use-contractkit/actions/runs/2333118394).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -336,10 +341,27 @@ use-contrackit uses Tailwind for styling, to use the modal in dark mode simply a
...
+```sh
+cd packages/use-contractkit
+yarn dev
+```
+
+4. Lastly, run the `example` app:
+
+```sh
+cd packages/example
+yarn dev
```
Comment (aaronmgdr) [inline readme.md:363]: if you run `yarn dev` from root it will run both at same time so so you dont need to run yarn dev in 2 browsers

Comment (crondinini) [reply readme.md:363]: When I run yarn dev from root, it tries to run wallet connect and it fails. :(

Comment (crondinini) [reply readme.md:363]: 🤔 Now if I run it, it works, but when I cloned it initially and followed the steps it didn't work. I'll check what's missing.

Comment (crondinini) [reply readme.md:363]: FYI, these are the errors:
```
example: wait  - compiling...
example: error - ./pages/_app.tsx:1:0
example: Module not found: Can't resolve '@celo-tools/use-contractkit/lib/styles.css'
example: > 1 | import '@celo-tools/use-contractkit/lib/styles.css';
example:   2 | import '../styles/global.css';
example:   3 |
example:   4 | import { Alfajores, ContractKitProvider } from '@celo-tools/use-contractkit';
example: https://nextjs.org/docs/messages/module-not-found
@celo-tools/use-contractkit: src/connectors/connectors.ts(17,8): error TS2307: Cannot find module '@celo/wallet-walletconnect-v1' or its corresponding type declarati
ons.
@celo-tools/use-contractkit: src/connectors/connectors.ts(420,34): error TS7006: Parameter 'error' implicitly has an 'any' type.
@celo-tools/use-contractkit: src/connectors/connectors.ts(420,41): error TS7006: Parameter 'session' implicitly has an 'any' type.
@celo-tools/use-contractkit: src/connectors/useWalletConnectConnector.ts(3,26): error TS2307: Cannot find module '@celo/wallet-walletconnect-v1' or its corresponding type declarations.
@celo-tools/use-contractkit: error Command failed with exit code 1.
@celo-tools/use-contractkit: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
@celo-tools/use-contractkit: error Command failed with exit code 1.
@celo-tools/use-contractkit: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run dev exited 1 in '@celo-tools/use-contractkit'
lerna WARN complete Waiting for 3 child processes to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
use-contractkit on  master via  v14.19.1 took 6s
```

Comment (crondinini) [reply readme.md]: Running `yarn build` between `yarn` and `yarn dev` works. So I put back the original instructions but left the new text as an alternative. Let me know what you think.

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**